pom.xml

```diff
@@ -2121,12 +2121,100 @@
+                                <repositoryType>PLUGIN</repositoryType>
+                            </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>org.apache.yetus</groupId>
+                                <artifactId>audience-annotations</artifactId>
+                                <version>0.8.0</version>
+                                <repositoryType>PLUGIN</repositoryType>
+                            </DynamicDependency>
                             <DynamicDependency>
                                 <groupId>io.airlift.drift</groupId>
                                 <artifactId>drift-javadoc</artifactId>
                                 <version>${dep.drift.version}</version>
                                 <repositoryType>PLUGIN</repositoryType>
                             </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>com.google.errorprone</groupId>
+                                <artifactId>error_prone_core</artifactId>
+                                <version>${dep.errorprone.version}</version>
+                                <repositoryType>MAIN</repositoryType>
+                            </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>org.junit.jupiter</groupId>
+                                <artifactId>junit-jupiter-engine</artifactId>
+                                <version>5.3.2</version> <!-- legacy version which Surefire seems to need for some reason -->
+                                <type>pom</type>
+                                <repositoryType>MAIN</repositoryType>
+                            </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>org.junit.jupiter</groupId>
+                                <artifactId>junit-jupiter-params</artifactId>
+                                <version>5.3.2</version> <!-- legacy version which Surefire seems to need for some reason -->
+                                <type>pom</type>
+                                <repositoryType>MAIN</repositoryType>
+                            </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>org.mockito</groupId>
+                                <artifactId>mockito-core</artifactId>
+                                <version>2.28.2</version> <!-- legacy version which Surefire seems to need for some reason -->
+                                <type>pom</type>
+                                <repositoryType>MAIN</repositoryType>
+                            </DynamicDependency>
+                            <DynamicDependency>
+                                <groupId>org.powermock</groupId>
+                                <artifactId>powermock-reflect</artifactId>
```

Comment:
What does the CI print if we don't add this.
I.e. if i introduce a new dynamic dependency (which obviously works in local devel), and run the build on the CI, how will I be informed I need to update this file?